report.html

```diff
@@ -0,0 +1,928 @@
+<!DOCTYPE html>
```

Comment:
The report file is supposed to be generated for each test run. These no need to commit it. Please remove it.